src/casp/cromwell/bq_ops/extract/inputs_extract.json

```diff
@@ -0,0 +1,8 @@
+{
+  "CASExtractBQ.extract.project_id": "dsp-cell-annotation-service",
+  "CASExtractBQ.extract.bq_dataset": "cas_benchmark_v1",
+  "CASExtractBQ.extract.extract_table_prefix": "kc_bench_v1",
+  "CASExtractBQ.bin_borders": [[0, 9], [10, 19], [20, 29], [30, 39], [40, 49], [50, 59], [60, 69], [70, 79], [80, 89], [90, 99]],
```

Comment:
It might be nice to have a "bin batch size" of 10, rather than specify the borders here.  It would be easier to optimize your bin sizes, and also not to forget bins (e.g. here there are really 399 bins I think but you're only extracting the first 100)